README.md

```diff
@@ -0,0 +1,71 @@
+# Project Name
+
+Awesome Books ES6 Modules
+
+# Description the project.
+
+In this project I made book list, while refactoring it to Javascript modules.
+
+## Built With
+
+- html/css/javascript ES6
+- following linters standards.
+- following github flows standards.
+
+## Getting Started
+
+**Awesome Books ES6 works on mobile and desktop using @Media queries .**
+
+---
+
+To get a local copy up and running follow these simple example steps.
+follow>>>>>>>>>
+
+### Setup
+
+to use the project Awesome_book_es6 please follow :
+
+> Open GitHub and go to my GitHub repository name: Awesome_book_es6 clone.
+> Click “Code” and copy the given URL.
+> click code and copy the URL.
+
+> click code and copy the URL.
+
+> Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+
+> Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+### Usage
+
+> open your directory Awesome_book_es6\*
+> to run the file :
+> \_use vs-code or any IDE to view line code .
+
+\_use you preferd browser to run the file .html to get project website.
+
+## Authors
+
+👤 **Edi Sipka**
+
+- GitHub: [@Edi-Sipka](https://github.com/edi-sipka)
+- Twitter: [@sipka_edii](https://twitter.com/sipka_edii)
+- LinkedIn: [Edi Sipka](https://www.linkedin.com/in/edi-%C5%A1ipka-5b681b202/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+thanks:
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] This link is broken, I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯